### PR TITLE
Vending machines

### LINKED ISSUE #263: venders

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR
todo:
- [x] Add all sprites
- [x] Fix sprite animation speeds
- [x] Make prototypes
- [x] Fill what can be filled

## Why / Balance
Resolves #263 

## Media
image_2023-12-03_15-17-56

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Drinks/.drinks_bottles.yml.swp
Comment (QuietlyWhisper) [inline Resources/Prototypes/Entities/Objects/Consumable/Drinks/.drinks_bottles.yml.swp:1]: swap file go brrr (I don't know why these keep coming up now)

File: Resources/Prototypes/_CM14/Catalog/VendingMachines/Inventories/chemistry.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- type: vendingMachineInventory
+  id: CMVendorChemistry
+  #startingInventory:
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Catalog/VendingMachines/Inventories/chemistry.yml:3]:
```suggestion
  #startingInventory: #todo cm14
```

File: Resources/Prototypes/_CM14/Catalog/VendingMachines/Inventories/cigs.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+- type: vendingMachineInventory
+  id: CMVendorCigs
+  startingInventory:
+    CigPackBlack: 50 #lucky strikes
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Catalog/VendingMachines/Inventories/cigs.yml:4]:
```suggestion
    CigPackBlack: 50 #lucky strikes #todo cm14
```

File: Resources/Prototypes/_CM14/Catalog/VendingMachines/Inventories/cola.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+- type: vendingMachineInventory
+  id: CMVendorCola
+  startingInventory:
+    DrinkColaCan: 10
+    # Souto drinks, All souto drinks, like all all. 10
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Catalog/VendingMachines/Inventories/cola.yml:5]:
```suggestion
    # TODO cm14 Souto drinks, All souto drinks, like all all. 10
```

File: Resources/Prototypes/_CM14/Catalog/VendingMachines/Inventories/marine_food.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+- type: vendingMachineInventory
+  id: CMVendorMarineFood
+  startingInventory:
+    # Prepared meal
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Catalog/VendingMachines/Inventories/marine_food.yml:4]:
```suggestion
    # Prepared meal todo cm14
```

File: Resources/Prototypes/_CM14/Catalog/VendingMachines/Inventories/marine_med.yml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+- type: vendingMachineInventory
+  id: CMVendorMarineMedical
+  #startingInventory:
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Catalog/VendingMachines/Inventories/marine_med.yml:3]:
```suggestion
  #startingInventory: todo cm14
```

File: Resources/Prototypes/_CM14/Catalog/VendingMachines/Inventories/medical.yml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+- type: vendingMachineInventory
+  id: CMVendorMedical
+  #startingInventory:
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Catalog/VendingMachines/Inventories/medical.yml:3]:
```suggestion
  #startingInventory: todo cm14
```

**Comment by Tunguso4ka** [comment]
Ready to review.
My opinion is that med vendors should be filled in medical PR.

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/.vending_machines.yml.swp
[the remark this answers is not in the document]
Comment (Tunguso4ka) [reply Resources/Prototypes/_CM14/Entities/Structures/Machines/.vending_machines.yml.swp:1]: oh.

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**